### PR TITLE
Replace Travis CI with Github actions

### PR DESCRIPTION
**Pull request by Gruppio**
**Why**
Github Actions are more reliable and still free for open source projects

**Changes**
None

**Tasks**
* [ ] Add relevant tests, if needed
* [ ] Add documentation, if needed
* [x] Update README, if needed
* [ ] Ensure that all the examples (as well as the demo) work properly

### CODE REVIEW

**Comment by Gruppio** [comment]
Test failure is not related to new CI

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+jobs:
+  build:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build
+      run: swift build
+    - name: Test
+      run: swift test
```
Comment (danyf90) [inline .github/workflows/build_and_test.yml:14]: [i] add newline

**Review by danyf90 — APPROVED** [review]

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (LorDisturbia) [inline .github/workflows/build_and_test.yml:14]:
```suggestion
      run: swift test

```

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
```
Comment (daquinoaldo) [inline .github/workflows/build_and_test.yml:10]:
```suggestion
    - uses: actions/checkout@v2
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 ![Katana](https://raw.githubusercontent.com/BendingSpoons/katana-swift/master/.github/Assets/katana_header.png)
 
 [![Twitter URL](https://img.shields.io/twitter/url/https/twitter.com/fold_left.svg?style=social&label=Follow%20katana_swift)](https://twitter.com/katana_swift)
-[![Build Status](https://travis-ci.org/BendingSpoons/katana-swift.svg?branch=master)](https://travis-ci.org/BendingSpoons/katana-swift)
+[![Build Status](https://github.com/BendingSpoons/katana-swift/workflows/Build%20and%20Test/badge.svg)](https://github.com/BendingSpoons/katana-swift/workflows/Build%20and%20Test/)
```
Comment (LorDisturbia) [inline README.md:5]: The workflows link doesn't work

Comment (Gruppio) [inline README.md:5]:
```suggestion
[![Build Status](https://github.com/BendingSpoons/katana-swift/workflows/Build%20and%20Test/badge.svg)](https://github.com/BendingSpoons/katana-swift/actions/workflows/build_and_test.yml)
```
This link should work once we will enable the Github Actions in this repo

**Pull request merged**